packages/react-components/src/components/SendBox.tsx

```diff
@@ -182,6 +215,16 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     setTextValue(newValue);
   };
 
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+    const fileUploads: ActiveFileUploads[] = props.activeFileUploads || [];
+    const latestError = fileUploads.filter((fileUpload) => fileUpload.errorMessage).pop();
+    if (latestError) {
+      return <Stack className={mergeStyles(errorBarStyle(theme))}>{latestError.errorMessage}</Stack>;
```

Comment:
Instead of a Stack can you use a fluent warning message bar here? https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar

Reply:
Done